Makefile

```diff
@@ -0,0 +1,254 @@
+# To make the document with the default settings:
+# > make
+#
+# To make with a specific format:
+# > make <format>
+#
+# To tar up a flat version of a specific format:
+# > make <format> tar
+#
+# Alex Drlica-Wagner: https://github.com/LSSTDESC/start_paper/issues/new?body=@kadrlica
+
+# Primary file names - avoid cookiecutter variables, to enable `make
+# upgrade` to cleanly over-write this Makefile...
+main ?= main
+#default=$(shell cat .metadata.json | grep 'default_format' | cut -d'"' -f4)
+outname ?= $(notdir $(shell pwd))
+
+#ifeq ($(default), {{ cookiecutter.default_format }})
+style ?= lsstdescnote
+#else
+# style=${default}
+#endif
+
+# mkauthlist doesn't recognize lsstdescnote as a style, so tell it to do something generic
+ifeq ($(style),lsstdescnote)
+mastyle = tex
+else
+mastyle = $(style)
+endif
+
+DESCTEX := desc-tex
+DESCTEXORIGIN := git@github.com:LSSTDESC/desc-tex.git
+
+localpip ?= F
+PIPOPTS = --upgrade-strategy only-if-needed
+ifeq ($(localpip),T)
+MKAUTHBIN := bin/mkauthlist
+MKAUTH := PYTHONPATH=$$(ls -d lib/python*/site-packages 2>/dev/null | head -n1):$$PYTHONPATH $(MKAUTHBIN)
+PIPOPTS += --install-option="--prefix=$(PWD)"
+GETMKAUTH := [ -e $(MKAUTHBIN) ] || pip install $(PIPOPTS) mkauthlist
+else
+MKAUTHBIN :=
+MKAUTH := mkauthlist 
+GETMKAUTH := pip list 2>/dev/null | grep -q mkauthlist || pip install $(PIPOPTS) mkauthlist
+endif
+
+# LATEX environment variables
+export TEXINPUTS:=./$(DESCTEX)/styles/:./tables/:
+export BSTINPUTS:=./$(DESCTEX)/bst/:
+export BIBINPUTS:=./$(DESCTEX)/bib/:
+
+# LaTeX journal class switcher flags
+# apj=\def\flag{apj}
+# apjl=\def
+# mnras=\def\flag{mnras}
+
+# Submission flags (these need some thought)
+# arxiv=\def\flag{emulateapj}
+# submit=${aastex}
+# draft=\def\linenums{\linenumbers}
+
+# Files to copy when making tarball
+tardir := tmp
+figdir ?= ./figures
+figures ?= $(figdir)/*.{png,jpg,pdf}
+tabdir ?= ./tables
+tables ?= $(tabdir)/*.tex
+styles ?= ./$(DESCTEX)/styles/*.{sty,cls}
+bibs ?= ./$(DESCTEX)/bib/*.bst
+source = $(main).{tex,bbl,bib} $(DESCTEX)/bib/lsstdesc.bib $(DESCTEX)/ack/*.tex authors.tex contributions.tex
+
+tarfiles = $(figures) $(tables) $(styles) $(bibs) $(source)
+
+# prefix for difference files
+DIFPRE ?= diff_
+# git repo branch to compare changes against
+BASEBRANCH ?= master
+# git repo branch we are compiling
+THISBRANCH := $(shell git branch --no-color 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/ \1/' | tr -d ' ')
+# temp file for the version to compare against
+DRAFT = $(main).$(BASEBRANCH)
+# base for difference files to generate
+DIFF = $(DIFPRE)$(THISBRANCH)--$(BASEBRANCH)
+
+
+maketargets := all authlist clean copy diff help $(main) update tar templates tidy touch
+.PHONY: $(maketargets)
+
+styleopts := aastex61 apj apjl emulateapj lsstdescnote mnras prd prl tex 
```

Comment:
Can we have "note" map to "lsstdescnote" style please? I thought `make note` was a nicer UX.